dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScopeTest.kt

```diff
@@ -1160,6 +1160,259 @@ internal class RumResourceScopeTest {
         assertThat(resultTiming).isEqualTo(null)
     }
 
+    @Test
```

Comment:
All these tests are not directly related with the `RumResourceScope` but more with the way we extract the timings from the attributes. That is the reason why I suggested a single responsibility component for that logic.